charts/stable/oauth2-proxy/README.md.gotmpl

```diff
@@ -143,3 +143,4 @@ helm install {{ template "chart.name" . }} {{ template "custom.helm.path" . }} -
 {{ template "custom.support" . }}
 
 {{ template "helm-docs.versionFooter" . }}
+{{ "" }}
```

Comment:
I got this diff by ran `./hack/gen-helm-docs.sh stable oauth2-proxy`.
Is it intentionally?

Reply:
That looks fine

Reply:
It's there to make sure there is a newline at the end of the file. Otherwise that somehow gets trimmed 😞